samples/shields/lmp90100_evb/rtd/sample.yaml

```diff
@@ -0,0 +1,7 @@
+sample:
+  name: LMP90100 EVB RTD
+tests:
+  sample.shields.lmp90100_evb.rtd:
+    harness: shield
+    tags: shield
+    depends_on: arduino_spi
```

Comment:
I think you should filter out the boards that already feature a built-in ADC.
I've faced same issue with BLE shield, and have actually not found a convenient way of doing it.
So I ended up using a whitelist...

Reply:
@erwango Thanks. I haved added a whitelist to `sample.yaml`.